Sources/Fuzzilli/FuzzIL/Analyzer.swift

```diff
@@ -168,6 +168,14 @@ struct ContextAnalyzer: Analyzer {
 
                 newContext.formUnion(contextStack.secondToTop)
             }
+            
+            // If we are in a loop, we don't want to propagate the switch context and vice versa.
```

Comment:
Can you add something like
`Otherwise, we could determine which type of break operations to emit. TODO consider building a generic mechanism for this if we need similar logic in other cases as well. For example, we could potentially add a instr.op.contextClosed list to generalize this behavior.`